js/lib/ui.js

```diff
@@ -73,6 +75,11 @@ elgg.ui.toggles = function(event) {
 			$elem.toggle();
 		}
 	});
+
+	$(document).trigger('elgg.ui.toggle', [{
+		$toggler: $this,
+		$target: $target
+	}]);
```

Comment:
I like this pattern. Why not add target to data and trigger the event on
$this?
On Nov 28, 2015 5:13 PM, "Steve Clay" <notifications@github.com> wrote:

> In js/lib/ui.js
> <https://github.com/Elgg/Elgg/pull/9159#discussion_r46081316>:
>
> > @@ -73,6 +75,11 @@ elgg.ui.toggles = function(event) {
> >  			$elem.toggle();
> >  		}
> >  	});
> > +
> > +	$(document).trigger('elgg.ui.toggle', [{
> > +		$toggler: $this,
> > +		$target: $target
> > +	}]);
>
> I could of course replace with with elgg.trigger_hook, and pass in the
> element/string instead of $-wrapped elements.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/9159/files#r46081316>.
>